docs/content/setup/docker.md

```diff
@@ -1,20 +1,91 @@
 # HedgeDoc Docker Image
 
-[![Try in PWD](https://cdn.rawgit.com/play-with-docker/stacks/cff22438/assets/images/button.png)](http://play-with-docker.com?stack=https://github.com/hedgedoc/container/raw/master/docker-compose.yml&stack_name=hedgedoc)
+!!! info "Requirements on your server"
+    - [Git](https://git-scm.com/)
+    - [Docker](https://docs.docker.com/get-docker/) 17.03.1 or higher
+    - [Docker Compose](https://docs.docker.com/compose/install/)
 
-## Debian-based version
+Official docker images are [available on quay.io](https://quay.io/repository/hedgedoc/hedgedoc).
+We currently only support the `amd64` architecture.
 
-[![Docker Repository on Quay](https://quay.io/repository/hedgedoc/hedgedoc/status "Docker Repository on Quay")](https://quay.io/repository/hedgedoc/hedgedoc)
 
-## Alpine-based version
+The easiest way to get started with HedgeDoc and Docker is using the following `docker-compose.yml`:
 
-[![Docker Repository on Quay](https://quay.io/repository/hedgedoc/hedgedoc/status "Docker Repository on Quay")](https://quay.io/repository/hedgedoc/hedgedoc)
+!!! warning
+    This is a minimal example to get started quickly and not intended for production use.
 
-The easiest way to setup HedgeDoc using docker are using the following three commands:
+```yaml
```

Comment:
While trying this, I could not upload an image…

Reply:
That's the reason why the original compose file had imgur by default.
Using local file uploads by default requires setting `CMD_DOMAIN`.

Reply:
I added `domain` and `urlAddPort` to both example configs, so uploads immediately work on localhost

Reply:
Can confirm: Docker works now with uploads.